packages/react/src/interop.ts

```diff
@@ -1,15 +1,20 @@
 // interop:
 import { AnyRouter } from '@trpc/server';
-import { createHooksInternalProxy } from './createTRPCReact';
-import { createHooksInternal } from './internals/createHooksInternal';
+import { CreateTRPCReact, createHooksInternalProxy } from './createTRPCReact';
+import {
+  CreateReactQueryHooks,
+  createHooksInternal,
+} from './internals/createHooksInternal';
 
 /**
  * @deprecated use `createTRPCReact` instead
  */
 export function createReactQueryHooks<
   TRouter extends AnyRouter,
   TSSRContext = unknown,
->() {
+>(): CreateReactQueryHooks<TRouter, TSSRContext> & {
+  proxy: CreateTRPCReact<TRouter, TSSRContext>;
+} {
```

Comment:
cc @colelawrence setting the return type here was what was needed to do reduce the type here